package.json

```diff
@@ -25,14 +25,14 @@
   ],
   "license": "Apache-2.0",
   "main": "scripts/build.js",
-  "private": "true",
+  "private": true,
   "repository": {
     "type": "git",
     "url": "git://github.com/koopjs/koopjs.github.io.git"
   },
   "scripts": {
     "build": "sitedown --build . --layout layout.html --watch .",
-    "start": "nodefront serve",
+    "start": "nodefront serve & npm run build",
```

Comment:
If you use `&` you may have to kill parallel processes explicitly using a pid and the `kill -9` command. [`npm-run-all`](https://github.com/mysticatea/npm-run-all) is a great utility for running npm tasks in parallel.